Flaky watchdog unit tests: test_3_daemon_with_watchdog/python_test_osqueryd (assertion errors)

While building and testing on RHEL6.5 (merged from master yesterday), I've been reliably getting an Assertion Error from `make test`.  

```
Running tests...
Test project /root/osquery/build/rhel6
    Start 1: osquery_tests
1/9 Test #1: osquery_tests ....................   Passed    0.36 sec
    Start 2: osquery_additional_tests
2/9 Test #2: osquery_additional_tests .........   Passed    5.53 sec
    Start 3: osquery_tables_tests
3/9 Test #3: osquery_tables_tests .............   Passed    0.14 sec
    Start 4: python_test_osqueryi
4/9 Test #4: python_test_osqueryi .............   Passed    7.89 sec
    Start 5: python_test_osqueryd
5/9 Test #5: python_test_osqueryd .............***Failed    1.71 sec
.I0619 20:21:58.394845 24438 config.cpp:129] Verbose logging enabled by config option
I0619 20:21:58.396441 24439 events.cpp:429] Starting event publisher run loop: inotify
I0619 20:21:58.396474 24440 events.cpp:429] Starting event publisher run loop: udev
.F
======================================================================
FAIL: test_3_daemon_with_watchdog (__main__.DaemonTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/root/osquery/tools/tests/test_osqueryd.py", line 62, in test_3_daemon_with_watchdog
    self.assertTrue(len(children) > 0)
AssertionError

----------------------------------------------------------------------
Ran 3 tests in 0.614s

FAILED (failures=1)

    Start 6: python_test_modules
6/9 Test #6: python_test_modules ..............   Passed    3.28 sec
    Start 7: python_test_extensions
7/9 Test #7: python_test_extensions ...........   Passed    6.00 sec
    Start 8: python_test_additional
8/9 Test #8: python_test_additional ...........   Passed    1.32 sec
    Start 9: python_test_example_queries
9/9 Test #9: python_test_example_queries ......   Passed    3.27 sec

89% tests passed, 1 tests failed out of 9

Total Test time (real) =  29.52 sec

The following tests FAILED:
	  5 - python_test_osqueryd (Failed)
Errors while running CTest
make[1]: *** [test] Error 8
make: *** [test] Error 2
```

--- a/tools/tests/test_base.py
+++ b/tools/tests/test_base.py
@@ -184,12 +184,12 @@ class ProcRunner(object):
         try:
             proc = psutil.Process(pid=self.proc.pid)
             delay = 0
-            while len(proc.get_children()) == 0:
+            while len(proc.children()) == 0:
                 if delay > max_interval:
                     return []
                 time.sleep(self.interval)
                 delay += self.interval
-            return [p.pid for p in proc.get_children()]
+            return [p.pid for p in proc.children()]
         except:
             pass
         return []

